### PR TITLE
Update to Bionic Beaver & cleanup

### PR DESCRIPTION
**Pull request by moonglum**
* [x] Update to Bionic Beaver
* [x] Remove Clipboard Forwarding, because I don't use it
* [x] Remove Go, ruby-switch, awscli, awsebcli & hub, because I don't use it
* [x] Remove vim 8 PPA, because vim 8 is part of Bionic Beaver
* [x] Remove git, htop, python3, tmux, wget from the install, because they are installed by default
* [x] Update OpenJDK
* [x] Vim-provements #73 (use [ale](https://github.com/w0rp/ale) with solargraph)
* [x] Add Github & Gitlab to known hosts
* [x] Install apt package manpages-dev
* [x] Try out [Input](http://input.fontbureau.com) as the new font
* [x] Install and use fzf
* [x] Review tmux config & create a nicer bar
* [x] Install and use weechat
* [x] Use [fish](https://fishshell.com) instead of ZSH with a better prompt #75

Bugs:

* GPG Forwarding doesn't work right now 😿 I deactivated signing... #83

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -18,5 +17,6 @@ Vagrant.configure("2") do |config|
   config.vm.provision "ansible_local" do |ansible|
     ansible.playbook = "ansible/playbook.yml"
     ansible.compatibility_mode = "2.0"
+    ansible.install_mode = "pip"
```
Comment (moonglum) [inline Vagrantfile]: The `default` install mode doesn't work anymore, it throws errors for missing packages.

File: ansible/playbook.yml [hunk 2]
```diff
@@ -6,51 +6,34 @@
...
     apt_keys:
-      A14F4FCA: 'http://packages.erlang-solutions.com/ubuntu/erlang_solutions.asc'
       68576280: 'https://deb.nodesource.com/gpgkey/nodesource.gpg.key'
       0EBFCD88: 'https://download.docker.com/linux/ubuntu/gpg'
     apt_repositories:
-      - 'deb https://packages.erlang-solutions.com/ubuntu xenial contrib'
-      - 'deb https://deb.nodesource.com/node_{{ nodejs_version }} xenial main'
-      - 'deb [arch=amd64] https://download.docker.com/linux/ubuntu xenial stable'
+      - 'deb https://deb.nodesource.com/node_{{ nodejs_version }} bionic main'
+      # TODO: Replace with bionic when it is available
```
Comment (moonglum) [inline ansible/playbook.yml]: This is some weird shit... The repo exists, but it doesn't contain docker-ce...

File: ansible/playbook.yml [hunk 3]
```diff
@@ -219,8 +205,8 @@
...
       become: true
 
-    - name: Install go packages
-      command: /usr/lib/go-{{ go_version }}/bin/go get github.com/{{ item }}
-      args:
-        creates: /home/vagrant/go/bin/{{ item }}
-      with_items: "{{ go_packages }}"
+    - name: Download fzf
+      git: repo=https://github.com/junegunn/fzf.git dest=/home/vagrant/.fzf
+    # TODO: Only do this if fzf was updated
```
Comment (moonglum) [inline ansible/playbook.yml]: I guess this is not a huge problem, because it is pretty quick

File: Vagrantfile (hunk 1 above)
Comment (bitboxer) [reply Vagrantfile]: Yes, I had the same problem 👍

File: ansible/playbook.yml (hunk 2 above)
Comment (bitboxer) [reply ansible/playbook.yml]: It is empty right now, I am using the repo for the previous debian version til this is fixed.

**Comment by bitboxer** [comment]
I highly advise to add all tools you need to the list of packages, even if they installed by default. You never know when they change that. I had so many docker container problems because someone in the team changed the container to another one and imagemagick disappeared 😭 .

But otherwise congrats for doing the switch. Hope you like it as much as I do.

File: ansible/playbook.yml (hunk 2 above)
Comment (moonglum) [reply ansible/playbook.yml]: Yep, that's what I do as well 😄

**Comment by bascht** [comment]
@moonglum Re the non-working GPG forward: did you try to connect via `vagrant ssh -- -o ExitOnForwardFailure=yes`?

**Comment by bascht** [comment]
Vimprovements. I salute your, sir!

:tophat:
:arrow_up_down:

**Comment by moonglum** [comment]
🙇

**Pull request merged**